onnxruntime/core/providers/cpu/tensor/slice_helper.h

```diff
@@ -91,7 +92,8 @@ inline Status PrepareForComputeHelper(const gsl::span<const int64_t>& raw_starts
   // Iterate through the provided axes and override the start/end/steps ranges
   using AxesSet = InlinedHashSet<int64_t>;
   const auto axes_count = axes.size();
-  AxesSet unique_axes(axes_count);
+  AxesSet unique_axes;
+  unique_axes.reserve(axes_count);
```

Comment:
Why did these need to be changed?

Reply:
The constructor takes a number of buckets. `reserve()` takes a number of items.